usercommandmgmt/info.json

```
@@ -0,0 +1,9 @@
+{
+    "AUTHOR" : "UBC Anime Club",
```

Author is a list of strings, see [this](https://docs.discord.red/en/stable/guide_publish_cogs.html#info-json-format).
```suggestion
    "author" : ["UBC Anime Club"],
```